server/protocols/jmap-rfc-8621-integration-tests/jmap-rfc-8621-integration-tests-common/src/main/scala/org/apache/james/jmap/rfc8621/contract/VacationResponseGetMethodContract.scala

```diff
@@ -145,8 +145,8 @@ trait VacationResponseGetMethodContract {
          |        {
          |          "id":"singleton",
          |          "isEnabled": true,
-         |          "fromDate": "2014-09-30T14:10:00+02:00",
-         |          "toDate": "2016-04-15T11:56:32.224+07:00[Asia/Vientiane]",
+         |          "fromDate": "2014-09-30T12:10:00Z[UTC]",
```

Comment:
Should be 2014-09-30T12:10:00Z (without [UTC]) as far as I understood.

Reply:
Seems so... Might need to use a custom DateTimeFormatter then I guess in the Serializer, will check it out